public/usage-examples/graphics/draw_rectangle/draw_rectangle-1-staircase-pattern.py

```diff
@@ -0,0 +1,21 @@
+from splashkit import *
+import random
+
+open_window("Draw Rectangle", 800, 600)
+clear_screen(color_white())
+
+for i in range(21):
+    
+    x = i * 40 - 40
+    y = 600 - i * 30
+
+    random_color = rgb_color(
+        random.randint(0, 255), random.randint(0, 255), random.randint(0, 255)   
```

Comment:
Then here use - 
https://deploy-preview-272--splashkit-io.netlify.app/api/utilities/#rnd-3

It would become `rnd_int(255), rnd_int(255), rnd_int(255)`